tests/test-cast-to-overflow.c

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
This empty line is not necessary and I don't see it in other files.
Could you remove it?

Reply:
Yes.